@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -552,7 +552,7 @@ export class VisibleTileSet {
                 // Update the visible area of the tile. This is used for those tiles that are
                 // currently loaded and are waiting to be decoded to sort the jobs by area.
                 tile.visibleArea = tileEntry.area;
-                tile.setElevation(tileEntry.minElevation, tileEntry.maxElevation);
+                tile.elevationRange = tileEntry;
```

Comment:
This will mutate the elevationRange object, probably not intended. You want only min/maxElevation.

Reply:
Yes, this is wrong, thanks for spotting it.